openslides/core/static/js/core/site.js

```diff
@@ -351,6 +351,64 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+.provider('MultiselectFilterProvider', [
```

Comment:
Why do you use a provider and not a factory (or a service)? A provider is useful if we want to configure something in the pre-run phase of Angular. Also the provider should not called Provider, see [here] (https://docs.angularjs.org/guide/providers#provider-recipe).